rosidl_generator_py/rosidl_generator_py/__init__.py

```diff
@@ -12,10 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .generate_py_impl import generate_py
-from .import_type_support_impl import import_type_support
+import logging
+import traceback
+
+__all__ = ['import_type_support']
 
-__all__ = [
-    'generate_py',
-    'import_type_support',
-]
+try:
+    from .generate_py_impl import generate_py
+    __all__.append('generate_py')
+except ImportError:
```

Comment:
You mean that traceback is not imported? Fixed in https://github.com/ros2/rosidl/commit/ff356c12ef1ce0d82fb9ef283d1407c1a3a30aba